packages/dev/core/src/Behaviors/Meshes/pointerDragBehavior.ts

```diff
@@ -49,11 +49,13 @@ export class PointerDragBehavior implements Behavior<AbstractMesh> {
      * Get or set the currentDraggingPointerId
      * @deprecated Please use currentDraggingPointerId instead
      */
+    // eslint-disable-next-line @typescript-eslint/naming-convention

```

Comment:
We do this for existing public APIs that don't follow the naming conventions, but for new stuff it should be enforced, is that right?

Reply:
correct. This is just to maintain back-compat. We should not use this disable flag after this is merged.